web/src/api/getMeta.ts

```diff
@@ -29,8 +29,9 @@ export async function getMeta(): Promise<Meta> {
 
 export function useMeta(): Meta {
   return (
-    useQuery<Meta>([QUERY_KEYS.META], async () => getMeta(), {
-      suspense: true,
+    useSuspenseQuery<Meta>({
+      queryKey: [QUERY_KEYS.META],
+      queryFn: async () => getMeta(),
     }).data ?? {}
```

Comment:
I think we can remove the `.data ?? {}` part here as data should always be defined when using suspense.